docs/resources/css/dark.css

```diff
@@ -1275,7 +1275,7 @@
 
     .wy-body-for-nav {
         background-image: initial;
-        background-color: rgb(26, 28, 29);
+        background-color: rgb(24, 26, 27);
```

Comment:
How did you come up with this new value?

Reply:
Got it from the bottom half with Inspect Element; it also has a better contrast with the main text block. Several members of the Python Discord helped decide on it.